nixos/modules/programs/x2goserver.nix

```diff
@@ -0,0 +1,121 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.x2goserver;
+
+  defaults = {
+    log = { "loglevel" = cfg.logLevel; };
+    x2goagent = { "port_randomization" = "pure-random"; };
+    superenicer = { "enable" = "no"; };
+    telekinesis = { "enable" = "no"; };
```

Comment:
Why are these defaults set?

Reply:
These are the defaults set in the configuration file included in the source distribution. I'll take them out, and see if anything breaks.

Reply:
nothing broke.